scripts/setup_wasi.sh

```diff
@@ -54,7 +54,7 @@ fi
 # Download wasi-sdk
 WASI_SDK_INSTALL_DIR="$TOOL_INSTALL_DIR/wasi-sdk-$WASI_SDK_VERSION_FULL"
 if [ ! -d $WASI_SDK_INSTALL_DIR ]; then
-    WASI_SDK_TARBALL="wasi-sdk-$WASI_SDK_VERSION_FULL-$SYS_OS.tar.gz"
+    WASI_SDK_TARBALL="wasi-sdk-$WASI_SDK_VERSION_FULL-$SYS_ARCH-$SYS_OS.tar.gz"
     WASI_SDK_INSTALL_URL="https://github.com/WebAssembly/wasi-sdk/releases/download/wasi-sdk-$WASI_SDK_VERSION/$WASI_SDK_TARBALL"
     OUTFILE="/tmp/$WASI_SDK_TARBALL"
```

Comment:
Would it work if OUTFILE changed to a generic versioned filename? That way when it extracts you don't need to worry about renaming

Reply:
I tried it, but it still changes back to that name with OS and ARCH values after extraction. So, I think renaming is the only option.

Reply:
Ah, gotcha. Thanks for looking into it!